js/lib/strings_utils.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
```

Comment:
Wanted to get the ball rolling on a strings library. Will flesh this out once this PR goes through in a dedicated PR